### PR TITLE
Created Italian translation

### PR DESCRIPTION
**Pull request by marcostudios**

### CODE REVIEW

**Comment by MadLittleMods** [comment]
@omame Can you take a look at this?

File: it.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 	"%d unread message": {
-		"one": "%d unread message",
-		"other": "%d unread messages"
+		"one": "%d messaggio non letto",
+		"other": "%d messaggi non letti"
 	},
 	"Unread messages in %%s, %%s and one other": {
-		"one": "Unread messages in %%s, %%s and one other",
-		"other": "Unread messages in %%s, %%s and %d others"
+		"one": "Messaggio non letto in %%s, %%s e un altro canale",
```
Comment (omame) [inline it.js]:
```

File: it.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-	"Unread messages from %s": "Unread messages from %s",
-	"Unread messages from %s and %s": "Unread messages from %s and %s",
+	"Unread messages in %s": "Messaggi non letti in %s",
+	"Unread messages in %s and %s": "Messaggi non letti in %s e %s",
+	"Unread messages from %s": "Messaggi non letti da %s",
+	"Unread messages from %s and %s": "Messaggi non letti da %s e %s",
 	"Unread messages from %%s, %%s and one other": {
-		"one": "Unread messages from %%s, %%s and one other",
-		"other": "Unread messages from %%s, %%s and %d others"
+		"one": "Messaggio non letto da %%s, %%s e un altro utente",
```
Comment (omame) [inline it.js]:
```
"one": "Messaggio non letto da %%s, %%s ed un altro utente",
```
The conjunction `e` becomes `ed` when the next word starts with a vowel. It's good grammar and many Italians makes this mistake, which makes this appear more professional.

File: it.js (hunk 1 above)
Comment (MadLittleMods) [inline it.js]: Correct

**Comment by omame** [comment]
@marcostudios  Can you also please fix [line 7](https://github.com/gitterHQ/gitter-translations/pull/25#discussion_r54231907)?
After that it's :+1: for me.

PS: I swear I'm not a grammar nazi. :stuck_out_tongue_winking_eye:

**Comment by marcostudios** [comment]
Sure :+1:

**Pull request merged**